stdlib/public/core/UnicodeScalarProperties.swift

```diff
@@ -519,35 +519,35 @@ extension Unicode.Scalar.Properties {
     return _hasBinaryProperty(__swift_stdlib_UCHAR_CASE_IGNORABLE)
   }
 
-  /// A Boolean property indicating whether the scalar is one whose normalized
-  /// form is not stable under a `toLowercase` mapping.
+  /// A Boolean property indicating whether the scalar's `lowercaseMapping`
+  /// differs from the NFD normalized form of the scalar itself.
   ///
   /// This property corresponds to the `Changes_When_Lowercased` property in the
   /// [Unicode Standard](http://www.unicode.org/versions/latest/).
   public var changesWhenLowercased: Bool {
     return _hasBinaryProperty(__swift_stdlib_UCHAR_CHANGES_WHEN_LOWERCASED)
   }
 
-  /// A Boolean property indicating whether the scalar is one whose normalized
-  /// form is not stable under a `toUppercase` mapping.
+  /// A Boolean property indicating whether the scalar's `uppercaseMapping`
+  /// differs from the NFD normalized form of the scalar itself.
   ///
   /// This property corresponds to the `Changes_When_Uppercased` property in the
   /// [Unicode Standard](http://www.unicode.org/versions/latest/).
   public var changesWhenUppercased: Bool {
     return _hasBinaryProperty(__swift_stdlib_UCHAR_CHANGES_WHEN_UPPERCASED)
   }
 
-  /// A Boolean property indicating whether the scalar is one whose normalized
-  /// form is not stable under a `toTitlecase` mapping.
+  /// A Boolean property indicating whether the scalar's `titlecaseMapping`
+  /// differs from the NFD normalized form of the scalar itself.
   ///
   /// This property corresponds to the `Changes_When_Titlecased` property in the
   /// [Unicode Standard](http://www.unicode.org/versions/latest/).
   public var changesWhenTitlecased: Bool {
     return _hasBinaryProperty(__swift_stdlib_UCHAR_CHANGES_WHEN_TITLECASED)
   }
 
-  /// A Boolean property indicating whether the scalar is one whose normalized
-  /// form is not stable under case folding.
+  /// A Boolean property indicating whether the scalar is one that is not
+  /// identical to its case-fold mapping.
```

Comment:
Can we validate that the "normalized" word is unnecessary? There are many scalars which are not in any normal form, so we'd want to make sure this property gives the equivalent answer as the normalized one.

Reply:
Yeah, this one was my attempt to reword it in such a way that it avoided the fact that we didn't have a `casefoldMapping` property like we have the others, but that piece got lost here. Your suggestion above made it easier to reword this to be parallel to those, so I just did that instead.